common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -875,8 +875,7 @@ protected AccountRecord getCachedAccountRecord(
                     "No accounts found for clientId ["
                             + clientId
                             + ", "
-                            + "]",
-                    null
+                            + "]"
```

Comment:
We can probably get rid of this whole Logger.info block.
As same info is getting logged again in line 880

Reply:
One will get filtered out if PII is turned off. Is that okay?